internal/clients/cfenvironment/client.go

```diff
@@ -90,33 +90,35 @@ func GetConnectionDetails(instance *provisioningclient.BusinessEnvironmentInstan
 	if instance == nil {
 		return managed.ConnectionDetails{}, nil
 	}
-	var cflabels cfEnvironmentLabels
-	var label string
+	cflabels := make(map[string]string)
+	var label []byte
 	if instance.Labels != nil {
-		label = *instance.Labels
+		label = []byte(*instance.Labels)
 	}
-	if err := json.Unmarshal([]byte(label), &cflabels); err != nil {
+	if err := json.Unmarshal(label, &cflabels); err != nil {
 		return managed.ConnectionDetails{}, err
 	}
 	details := managed.ConnectionDetails{
-		v1alpha1.ResourceRaw: []byte(label),
+		v1alpha1.ResourceRaw: label,
 	}
 
-	if cflabels.OrgName != nil {
-		details[v1alpha1.ResourceOrgName] = []byte(*cflabels.OrgName)
+	if orgName, ok := cflabels["Org Name"]; ok {
+		details[v1alpha1.ResourceOrgName] = []byte(orgName)
 	}
-	if cflabels.OrgId != nil {
-		details[v1alpha1.ResourceOrgId] = []byte(*cflabels.OrgId)
+
+	for _, orgIdKey := range []string{"Org ID:", "Org ID"} {
```

Comment:
nit: this way the old format take precedence over the new format. I suggest we do it the other way round just to be safe. If both combinations are available, use the newer one (without `:`)